vignettes/Introduction_To_BiocParallel.Rmd

```diff
@@ -0,0 +1,961 @@
+---
+title: "**Introduction to *BiocParallel***"
+author: "Phyllis Atieno"
+date: "Compiled: 18 October, 2022"
+package: BiocParallel
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+  
+---
+
+Introduction
+============
+
+Numerous approaches are available for parallel computing in R. The CRAN Task View for
+high performance and parallel computing provides useful high-level summaries and [package
+categorization.](http://cran.r-project.org/web/views/HighPerformanceComputing.html)
+Most Task View packages cite or identify one or more of **snow** , **Rmpi**, **multicore** or **foreach** as
+relevant parallelization infrastructure. Direct support in *R* for *parallel* computing started
+with release 2.14.0 with inclusion of the parallel package which contains modified versions of
+*multicore* and *snow* .
```

Comment:
Packages should be treated consistently -- linked to CRAN, same font -- but here they are now italic whereas previously they were bold. I do not know what the appropriate style is, but I would imagine it is whatever the BiocStyle package implements for the LaTeX macro `\CRANpkg{}`.